AI_Model.py

```diff
@@ -0,0 +1,42 @@
+from transformers import AutoModelForCausalLM, AutoProcessor
+import numpy as np
+from PIL import Image
+
+class AI_Model:
```

Comment:
Imena klasa se pisu u formatu ImeKlase a funkcije se pisu u formatu ime_funkcije. Ispraviti to u svim fajlovima